README.md

```diff
@@ -1,8 +1,15 @@
-# AnitaB.org Open Source Documentation
+# open-source-teams :wave:
+Place to document everything about our AnitaB.org Open Source teams
 
-This is the place to document everything about our AnitaB.org Open Source community.
+# Hello :eyeglasses: :v: 
+# [About AnitaB.org](https://anitab.org/)
+At AnitaB.org, we’re committed to increasing the influence of women on all aspects of technology. Our Local community expands our efforts globally to help individuals all over the world — especially those who are considering or currently pursuing technical careers — can access the resources they need to reach their highest potential.
+
+Members of the global AnitaB.org Local community network organize events and provide one another with resources to navigate careers in tech. They organize valuable meetups, codeathons, and one-day HopperX1 events modeled after the Grace Hopper Celebration.
```

Comment:
this can be added as part of another issue to add information introducing this community on README